### DISCUSSION TITLE
Is there another way to apply classes?

### DISCUSSION THREAD

**Discussion by BelkinMax** [tailwindlabs/tailwindcss#4918] [Help]
Hi guys 👋

This is the first time I use Tailwind in my project.

I really like this library, but I think that its weak part is that it makes my HTML grow wide and makes it kind of unreadable.

Using dynamic classes in Vue solves that, but it creates other problems. The first is that the script area is not a suitable place for CSS classes, and the second is that there is no way to use tailwind snippets in the script block.

Any idea how could it possibly be solved, or am I just procrastinating? 😃

**Comment by BelkinMax** [comment]
Just found that `@apply` is what I needed..

**Reply by denik1981** [reply]
A little adviced here, if it happens you are using webpack as your build tol, put all TW imports declarations outside the files where you are extracting your classes with @apply. Having them in the same file slows down the build process a LOT. You will know what I mean.
